### ISSUE TITLE
Error building with Gradle 5.0

### ISSUE THREAD

**Issue by vayan** [arturdm/jacoco-android-gradle-plugin#54]
I have an issue building my project with the latest [5.0 update](https://docs.gradle.org/5.0/release-notes.html) from Gradle 

```kotlin
dependencies {
    classpath("com.dicedmelon.gradle:jacoco-android:0.1.3") {
        exclude(group = "org.codehaus.groovy", module = "groovy-all")
    }
}
```

```
org.gradle.api.ProjectConfigurationException: A problem occurred configuring project 'foo'.
Caused by: groovy.lang.MissingPropertyException: No such property: dir for class: org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection
	at com.dicedmelon.gradle.jacoco.android.JacocoAndroidPlugin.logTaskAdded(JacocoAndroidPlugin.groovy:103)
	at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:479)
	at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:191)
	at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:78)
	at com.dicedmelon.gradle.jacoco.android.JacocoAndroidPlugin$_apply_closure1.doCall
```

**Comment by tobiasKaminsky** [comment]
This is also happening for me with 4.x.
Strange is that it is only happening in a subproject of my library, but nowhere else…

**Comment by cleanlavabo** [comment]
How about this ticket?

**Comment by brianguertin** [comment]
Looks like the offending code is just a log statement:

```
logger.info("  classDirectories: $reportTask.classDirectories.dir.path")
```

So, an easy work around would be to skip logging. Of course, it would be better would to not use `internal` APIs.

**Comment by mrfatstrat** [comment]
+1 any plans for Gradle 5.x support soon?

**Comment by tobiasKaminsky** [comment]
Strange thing is that in AS 3.3, e.g. either sync gradle, or even execute assemble in grade tab, it does not work:
```
* What went wrong:
A problem occurred configuring root project 'android-library'.
> groovy.lang.MissingPropertyException: No such property: dir for class: org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```

But when I use "./gradlew assemble" or any other task from console, I do not get an error and it builds just fine.

**Comment by stephanenicolas** [comment]
Same here, we are actually removing the plugin from a project because it's not updated for gradle 5.

**Comment by arturdm** [comment]
Fixed in [0.1.4](https://github.com/arturdm/jacoco-android-gradle-plugin/releases/tag/jacoco-android-release-0.1.4) release.

**Comment by Taylor004** [comment]
how to fixed that error...I don't know hot to fix my project....same build error
